### PR TITLE
Create 2017-07-18-ember-best-practices-form-components.md

### PR DESCRIPTION
**Pull request by nicomihalich**
Rendered: https://github.com/DockYard/reefpoints/blob/2b35992cd71e10e2c880a49cc96ab3bd403c99d5/source/posts/2017-06-16-ember-best-practices-form-components.md

### CODE REVIEW

File: source/posts/2017-06-16-ember-best-practices-form-components.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: "Ember Best Practices: Reducing CRUD complexity with components"
+social: true
+author: Nico Mihalich
+github: nicomihalich
+summary: "Reduce your Ember CRUD code with form components"
+published: true
+tags: Ember, Best Practices
```
Comment (brzpegasus) [inline source/posts/2017-06-16-ember-best-practices-form-components.md]: Can you also tag with `Engineering`?

File: source/posts/2017-06-16-ember-best-practices-form-components.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  </button>
+{{/widget-form}}
+```
+
+And edit like this:
+
+```handlebars
+{{#widget-form model=model}}
+  <button type="submit" {{action "save"}}>
+    Create Widget
```
Comment (brzpegasus) [inline source/posts/2017-06-16-ember-best-practices-form-components.md]: `Update Widget`?

**Comment by jesskry** [comment]
I'd like for us to publish this Thursday, June 29.

File: source/posts/2017-06-16-ember-best-practices-form-components.md (hunk 2 above)
Comment (nicomihalich) [inline source/posts/2017-06-16-ember-best-practices-form-components.md]: Changed this one to update https://github.com/DockYard/reefpoints/pull/581/files#diff-b8f97fe4bce5e87e0d9d1056fc3aa1dfR84

**Pull request merged**